### ISSUE TITLE
Issue with "How to structure a web form": link has incorrectly-assigned class.

### ISSUE THREAD

**Issue by EricRa** [mdn/content#14448]
MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form

#### What information was incorrect, unhelpful, or incomplete?
The "aria-label" link in the sentence below is part of the class "page-not-created".   However, the page it links to is complete, and this class label should be removed so that the CSS correctly displays this as an active, working link.

"Titles being read aloud depend on the screen reader's settings, so it is more reliable to also include the [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute) attribute, which is always read by screen readers."

#### Specific section or headline?

[Multiple labels](https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form#multiple_labels)

#### What did you expect to see?

The display of the link makes it seem like the page is not yet written, but the linked page is actually accessible and appears complete.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/learn/forms/how_to_structure_a_web_form`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/forms/how_to_structure_a_web_form/index.md
* Last commit: https://github.com/mdn/content/commit/d653233894417910925b2fc10944e1676e9cca6e
* Document last modified: 2022-03-20T14:53:24.000Z

</details>

**Comment by sideshowbarker** [comment]
Thanks for catching this — I’ve opened #14491 with a fix.

After that fix is merged, this issue will automatically be marked as resolved, and closed.
